tests/testsuite/standard_lib.rs

```diff
@@ -104,14 +104,8 @@ fn setup() -> Setup {
                         // to ensure nothing is required.
                         // See https://github.com/rust-lang/wg-cargo-std-aware/issues/31
                         // for more information on this.
-                        //
-                        // FIXME: this is broken on x86_64-unknown-linux-gnu
-                        // due to https://github.com/rust-lang/rust/pull/124129,
-                        // because it requires lld in the sysroot. See
-                        // https://github.com/rust-lang/rust/issues/125246 for
-                        // more information.
-                        // args.push("--sysroot".to_string());
-                        // args.push("/path/to/nowhere".to_string());
+                        args.push("--sysroot".to_string());
```

Comment:
Because the hack predates the test `build_std_with_no_arg_for_core_only_target`, which really requires a target shared library from sysroot, for example `~/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-none/lib/`

Reply:
One way to fix this is also ensure that the `--target` passed in is rustc host target.

Reply:
Ah I see... well, ensuring that passed `--target` is the host target seems a good way to go.

Reply:
(By the way, thank you for your advice)